.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+<!-- CLICK "Preview" FOR INSTRUCTIONS IN A MORE READABLE FORMAT -->
+
+## Description
+Please explain what features the pull request implemenets and/or changes made.
+
+## Reason behind implementation
+Does this pull request link to an existing feature request or a bug report? If so please provide a link.
+
+## Are there any other branches related to this work?
+
```

Comment:
I would add something along these lines: "If there are any branches related to this work please provide the branch name and a link."